src/Libraries/RevitNodes/Elements/ModelCurve.cs

```diff
@@ -134,20 +131,35 @@ private void InitModelCurve(Autodesk.Revit.DB.Curve crv, bool makeReferenceCurve
         #region Private mutators
 
         /// <summary>
-        /// Set the curve internally.  Returns false if this method failed to set the curve
+        /// Set the plane and the curve internally.
         /// </summary>
-        /// <param name="c"></param>
-        /// <returns></returns>
-        private bool InternalSetSketchPlaneFromCurve(Autodesk.Revit.DB.Curve c)
+        private void InternalSetSketchPlaneFromCurve(Curve newCurve)
         {
             TransactionManager.Instance.EnsureInTransaction(Document);
 
-            // Infer the sketch plane
-            Autodesk.Revit.DB.Plane plane = CurveUtils.GetPlaneFromCurve(c, false);
+            Plane newPlane = CurveUtils.GetPlaneFromCurve(newCurve, false);
+            Plane oldPlane = InternalCurveElement.SketchPlane.GetPlane();
+
+            var angleBetweenPlanes = newPlane.Normal.AngleTo(oldPlane.Normal);
```

Comment:
@mjkkirschner , ha-ha... I see)))) actually, I don't know. It depends how small angle we want have between planes. Before we used  System.Math.PI / 1800.0 * System.Math.PI / 1800.0. Which I guess is 0.01 of radian, if I'm correct.